packaging/nuget/packaging.py

```diff
@@ -523,74 +298,80 @@ def xlayout (self):
+             'build/native/lib/v120/x64/Release/librdkafkacpp.lib', 'build/native/lib/win7/x64/win7-x64-Release/v120/librdkafkacpp.lib'],
+
+            [{'arch': 'x86', 'plat': 'win7', 'fname_startswith': 'msvcr120.zip'}, 'msvcr120.dll', 'runtimes/win7-x86/native/msvcr120.dll'],
+            [{'arch': 'x86', 'plat': 'win7', 'fname_startswith': 'librdkafka.redist'}, 'build/native/bin/v120/Win32/Release/librdkafka.dll', 'runtimes/win7-x86/native/librdkafka.dll'],
+            [{'arch': 'x86', 'plat': 'win7', 'fname_startswith': 'librdkafka.redist'}, 'build/native/bin/v120/Win32/Release/librdkafkacpp.dll', 'runtimes/win7-x86/native/librdkafkacpp.dll'],
+            [{'arch': 'x86', 'plat': 'win7', 'fname_startswith': 'librdkafka.redist'}, 'build/native/bin/v120/Win32/Release/zlib.dll', 'runtimes/win7-x86/native/zlib.dll'],
+            [{'arch': 'x86', 'plat': 'win7', 'fname_startswith': 'librdkafka', 'fname_excludes': ['redist', 'symbols']}, 
+            'build/native/lib/v120/Win32/Release/librdkafka.lib', 'build/native/lib/win7/x86/win7-x86-Release/v120/librdkafka.lib'],
+            [{'arch': 'x86', 'plat': 'win7', 'fname_startswith': 'librdkafka', 'fname_excludes': ['redist', 'symbols']}, 
+            'build/native/lib/v120/Win32/Release/librdkafkacpp.lib', 'build/native/lib/win7/x86/win7-x86-Release/v120/librdkafkacpp.lib']
+        ]
+
+        for m in mappings:
+            attributes = m[0]
+            fname_startswith = attributes['fname_startswith']
+            del attributes['fname_startswith']
+            fname_excludes = []
+            if 'fname_excludes' in attributes:
+                fname_excludes = attributes['fname_excludes']
+                del attributes['fname_excludes']
+
+            artifact = None
+            for a in self.arts.artifacts:
+                found = True
+
+                for attr in attributes:
+                    if a.info[attr] != attributes[attr]:
+                        found = False
+                        break
+
+                if not a.fname.startswith(fname_startswith):
+                    found = False
+
+                for exclude in fname_excludes:
+                    if exclude in a.fname:
+                        found = False
+                        break
+
+                if found:
+                    artifact = a
+                    break
+
+            if artifact == None:
+                raise Exception('unable to find file in archive %s with tags %s that ends with "%s"' % (a.fname, str(attributes), fname_startswith))
+
+            outf = os.path.join(self.stpath, m[2])
+            member = m[1]
+            try:
+                zfile.ZFile.extract(artifact.lpath, member, outf)
+            except KeyError as e:
+                raise Exception('file not found in archive %s: %s. Files in archive are: %s' % (artifact.lpath, e, zfile.ZFile(artifact.lpath).getnames()))
 
-    def verify (self, path):
```

Comment:
What's the reason for removing the verifier?

Reply:
i thought it was superfluous - if an artifact isn't found during build, the program will exit.

Reply:
It is the only test we have for the package, and it is already written, so I believe it should stay in.

Reply:
ok, it's back in.